### PR TITLE
NERFS HULK BY MAKING IT SO THAT UPON APPLICATION THE HULK MUTATION IS AUTOMATICALLY REMOVED BETWEEN 3 TO 5 MINUTES AFTER APPLYING HULK, IT ALSO NOW STUNS FOR 5 SECONDS UPON HULK WEARING OFF THIS IS ALL IN THE TITLE BECAUSE APPARENTLY DESPITE GOING OVER THIS IN THE DESCRIPTION NO ONE CARES ENOUGH TO READ IT LOOKING AT YOU @PKPenguin12 AND @NikNakFlak

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Iamgoofball
experimental: Tampering with the genetic code has always been a risky and sometimes fruitless endeavor. It might not even last forever.
/:cl:

This has been a long time coming.

### CODE REVIEW

**Comment by Shadowlight213** [comment]
I ded

**Comment by Iamgoofball** [comment]
i haven't died to a hulk recently I just noticed that for the past week or so genetics can reliably get hulk 5-10 minutes in and now the entire station is hulk

people have tossed the idea around of making it function this way for ages so

**Review by Razharas — DISMISSED** [review]
Goof being shit again

**Comment by KorPhaeron** [comment]
Getting rads in the machine should give you temporary powers period

Using a syringe should give permanent powers

Making a syringe should cost uranium or something.

The powers are never going to be worth having and also balanced if you can give them to everyone for free. If you make all the powers not worth giving out then we may as well just remove the department.

**Comment by Iamgoofball** [comment]
the point is not to restrict who they are given to

it's so they aren't always just infinite hulk forever, you have to keep going back to your dealer to get juiced up, giving you a period of weakness

**Comment by Cebasiz** [comment]
worse than removing hulk, but better than leaving it alone

hulk is  greentide megacancer that really fucks over balance and really influences player psychology to be huge shitheads

**Comment by Iamgoofball** [comment]
hey can someone with expert knowledge of callbacks and shit help me make this work? it's not picking up the fuckin lose mutation stuff

File: code/datums/mutations.dm [hunk 1]
```diff
@@ -129,7 +129,8 @@
 	var/status = CANSTUN | CANWEAKEN | CANPARALYSE | CANPUSH
 	owner.status_flags &= ~status
 	owner.update_body_parts()
-
+	owner_dna = owner.dna // fukken callbacks
```
Comment (RemieRichards) [inline code/datums/mutations.dm]: this... isn't required.
just put owner.dna where you did in the callback

(if your code compiled, it'd be functionally identical to just putting the var in the callback, no need for a temporary var.)

Comment (Iamgoofball) [inline code/datums/mutations.dm]: i did that originally and got the same compile error so i tried this

Comment (RemieRichards) [inline code/datums/mutations.dm]: you quite literally could not have gotten "undefined var owner_dna" when you didn't write that before.

Comment (Iamgoofball) [inline code/datums/mutations.dm]: oh fuck

**Comment by Iamgoofball** [comment]
sadly still doesn't compile though, undefined proc

**Comment by XDTM** [comment]
Turns out goof is more convincing than me about removing hulk

**Comment by AnturK** [comment]
You want dna proc , you're using relative proc of mutation, you need to specify full path here. About the actual change, whatever i'm changing hulk anyway in the refactor so i guess this can be a stopgap.

**Comment by Niknakflak** [comment]
you should be repobanned if you keep putting meme images and no description of your changes in your PRs.

**Comment by Jazaen** [comment]
It's hulk only, right?

**Comment by PKPenguin321** [comment]
Undocumented change with weaken 5
Random no reasoning nerf
Can't even use addtimer

**Comment by Iamgoofball** [comment]
@Niknakflak 
What the fuck are you even talking about here? There was no meme shit in this PR aside from the title(which is accurate)

**Comment by Iamgoofball** [comment]
@PKPenguin321 did you even read the PR desc

**Comment by PKPenguin321** [comment]
Oh yeah misleading title too

**Comment by Iamgoofball** [comment]
fine fuck i guess its not good enough for you guys unless I baby you through everything

**Comment by PKPenguin321** [comment]
Reasoning is "this has been a long time coming" which isn't reasoning
No documentation for weaken 5

**Comment by PKPenguin321** [comment]
>let me name this pr something vague so people won't immediately see how controversial it is, maybe it'll get merged without anybody noticing!!!

>i-it was just a joke title guys

Scummy

**Comment by Iamgoofball** [comment]
>specifically states that it will make genetics mains mad
>think hm maybe i'll write a nice changelog description this time alluding to the fact that hulk now wears off

**Comment by PKPenguin321** [comment]
Nice call out in your title but point stands, scummy sneak thief

**Comment by Niknakflak** [comment]
Throw a little temper tantrum all you want.  It doesn't take away from the fact that you didn't give any description and had a shit title.  Pkp is right "It's been a long time coming" is NOT in the description so even your dumb renaming of your title is stupid.  Grow the fuck up

**Comment by Militaires** [comment]
*autistic screeching*

**Comment by Iamgoofball** [comment]
THE FUCKING DESCRIPTION STATED WHAT THE PR DID YOU ASSHOLE

IF I WANTED TO SNEAK THIS BY WHY WOULD I SPECIFICALLY CALL OUT GENETICS MAINS

**Comment by Iamgoofball** [comment]
also fyi before you bitch I'm simply complying with a headmins request :^)

**Comment by Niknakflak** [comment]
"It's been a long time coming" is not any kind of informative description. Stop changing your PRs to all caps and just do it right and professionally like a normal person.  Why do you always flip out?

**Comment by PKPenguin321** [comment]
Still no reasoning

**Comment by Militaires** [comment]
*apes in the distance screaming*

**Comment by Iamgoofball** [comment]
# reason invalid reopening PR

**Comment by PKPenguin321** [comment]
Here we go again!

**Pull request closed without merging**